django_fido/js/fido2.js

```diff
@@ -125,26 +125,23 @@ async function startFido2() {
         return // Silently skip if not on correct page.
 
     const submit_button = document.getElementById('submit-button')
-    // If is empty values, submit button reload page
-    submit_button.addEventListener('click', e => {
-        if (form.client_data.value === '' || form.credential_id.value === '' ||
-            form.authenticator_data.value === '' || form.signature.value === '')
-        {
+    if (submit_button) {
+        // If the values are empty values, submit button reload page
```

Comment:
Is there still reason to have this comment here?

Reply:
Probably not. I'll take care of it.

Reply:
Deleted.